fixtures/fixtures.js

```diff
@@ -0,0 +1,77 @@
+var studentData  = {
```

Comment:
What do you think about create bunch of data to have a possibility to make some 'mixes' ?

Reply:
Can you write a little bit more about it? Without any doubt we have to mix data from fixtures to get more unified data object.